integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/altinn/MessagePolling.java

```diff
@@ -120,7 +120,7 @@ public void checkForFiksMessages() {
 
     @Scheduled(fixedRate = 15000)
     public void checkForNewMessages() throws MessageException {
-        if (!properties.getFeature().isEnableDPO()) {
+        if (!properties.getFeature().isEnableDPO() || true) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "true" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1125)